### ISSUE TITLE
GPIOActor not working on Pi

### ISSUE THREAD

**Issue by avollkopf** [Manuel83/craftbeerpi4#15]
- With the standard installation the GPIO Actor is not working and is causing errors.
- As soon as the cbpi4-pt100x plugin is installed, the problem is gone.
- The plugin uses these commands during initialization to prevent the error:
		GPIO.setwarnings(False)
		GPIO.setmode(GPIO.BCM)
- Once a PT100 sensor is configured, the initilization takes place and the actor plugin is working fine

Proposal:
- Include the two lines of code into the gpioactor unless there is a better place for this
- Add RPi.GPIO to the requirements that it is being installed with cbpi
 
```
try:
    import RPi.GPIO as GPIO
except Exception:
    logger.error("Failed to load RPi.GPIO. Using Mock")
    MockRPi = MagicMock()
    modules = {
        "RPi": MockRPi,
        "RPi.GPIO": MockRPi.GPIO
    }
    patcher = patch.dict("sys.modules", modules)
    patcher.start()
    import RPi.GPIO as GPIO
GPIO.setwarnings(False)
GPIO.setmode(GPIO.BCM)
```

Later I will create a pull request for this proposal.

**Comment by MarcioMonego** [comment]
It doesn't appear that ignore the errors/warnings without to know what they is a good path.
Try send the warning error you got.
Cheers

**Comment by avollkopf** [comment]
Agreed. 

There is just a warning if left out:  

`RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
`
Will proved the request w/o this line or at least commented out

**Comment by avollkopf** [comment]
ok, this one would prevent to set the mode several times:

```
try:
    import RPi.GPIO as GPIO
except Exception:
    logger.error("Failed to load RPi.GPIO. Using Mock")
    MockRPi = MagicMock()
    modules = {
        "RPi": MockRPi,
        "RPi.GPIO": MockRPi.GPIO
    }
    patcher = patch.dict("sys.modules", modules)
    patcher.start()
    import RPi.GPIO as GPIO
mode=GPIO.getmode()
if (mode == None):
    GPIO.setmode(GPIO.BCM)
```

It does not prevent the warning which was probably the reason why the author of the max31865.py added it back in time

**Comment by avollkopf** [comment]
Fixed with pull request #16
